media-sound/mpd/mpd-0.21.8.recipe

```diff
@@ -139,34 +144,49 @@ BUILD_REQUIRES="
 	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
-	cmd:autoconf
+	setuptools_python3
 #	cmd:avahi_client$secondaryArchSuffix
-	cmd:awk
-	cmd:find
+	cmd:cmake
```

Comment:
are you sure about cmake

Reply:
It definetely searching for it. I was also surprised, but maybe it using cmake in pkg-config mode (it is possible, btw) to search for its deps.

Reply:
or there are third party deps which could require cmake, dunno